### PR TITLE
Create GsuiteUserPWReset.md

### PR DESCRIPTION
**Pull request by robeecaneda1**
these are the steps to reset a gsuite user's pw from the admin console

### CODE REVIEW

File: GSuite/GsuiteUserPWReset.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿[Gsuite Admin](https://admin.google.com)
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: There is no title.

File: GSuite/GsuiteUserPWReset.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+ - Click Users
+ - In the Users List, find the user.
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: If you click Users, do you then get the Users List? If so, delete this line

File: GSuite/GsuiteUserPWReset.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+ - Hover over the user you want to reset the password for and click "Reset password"
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: Keep it short.
Obviously we do not want to hover over the user we do NOT want to reset the password for

Ex: "- Hover User > Reset password

File: GSuite/GsuiteUserPWReset.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+ - Click Password > Reset Password
+ - Choose to automatically generate the password
+ - (Optional) Ask the user to change the new password the next time they sign in.
+ - Click Reset.
+ - Choose to email the password to the user, or click Done.
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: - Choose to email the password to the user
- Done

(We always want to email the user)

File: GSuite/GsuiteUserPWReset.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿[Gsuite Admin](https://admin.google.com)
+
+ - Click Users
+ - In the Users List, find the user.
+ - Hover over the user you want to reset the password for and click "Reset password"
+ - Click Password > Reset Password
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: I just tried it, looks like this line is also not necessary?

File: GSuite/GsuiteUserPWReset.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+ - Choose to automatically generate the password
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: This option is checked on by default, so also remove this line.

File: GSuite/GsuiteUserPWReset.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+ - (Optional) Ask the user to change the new password the next time they sign in.
```
Comment (Laoujin) [inline GSuite/GsuiteUserPWReset.md]: This open is checked by default, let's keep it that way. Remove this line.

File: GSuite/GsuiteUserPWReset.md (hunk 2 above)
Comment (robeecaneda1) [reply GSuite/GsuiteUserPWReset.md]: Yes, you will be redirected to all users. Okay, I'll update it to just "find the user"

File: GSuite/GsuiteUserPWReset.md (hunk 3 above)
Comment (robeecaneda1) [reply GSuite/GsuiteUserPWReset.md]: Copy

**Pull request merged**